src/snowcli/cli/nativeapp/commands.py

```diff
@@ -49,9 +49,9 @@ def app_init(
     ),
```

Comment:
The help string for name option was much larger in the main branch version, so while merging, i accepted this version - so please review it again, as it is a new tekst

Reply:
LGTM